### PR TITLE
Feature/ola thumbnail

### PR DESCRIPTION
**Pull request by mablancoalvarez**
Os dejamos para revisar el nuevo componente OlaThumbnail!

Cualquier cosa a modificar nos comentáis

### CODE REVIEW

File: src/OlaThumbnail.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        alt: 'string',
+        align: 'string',
+        'background-color': 'string',
+             
+    };
+
+    static defaultAttributes = {
+        alt: '',
+        align: 'center',
+        // width:'250px',
```
Comment (oscarotero) [inline src/OlaThumbnail.js:24]: Commented code should be removed

File: src/OlaThumbnail.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      `;
+    }
+    
+  
+    render() {
+        return this.renderMJML(`
+        <mj-section padding-bottom="0">
+        <mj-column>
+            <mj-image
+                css-class = "ola_thumbnail img"
```
Comment (oscarotero) [inline src/OlaThumbnail.js:44]: This attribute is not right. Should be `css-class="ola_thumbnail" (img is used only in the css selector, but it's not a class name).

File: src/OlaThumbnail.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                padding="0"
+                src="${this.getAttribute('src')}"
+                alt="${this.getAttribute('alt')}"
+                width="250px"
+                border="10"
+                // 'background-color'= "${this.getAttribute('background-color')})"
```
Comment (oscarotero) [inline src/OlaThumbnail.js:50]: Comented code :)

**Comment by griselmatosm** [comment]
We fixed the issues! Please leave us comments!

**Comment by mablancoalvarez** [comment]
We solved the total issues and we updated the new component with the new changes.

**Comment by oscarotero** [comment]
Thanks! 👍

**Pull request merged**